crates/mpz-ole/src/lib.rs

```diff
@@ -0,0 +1,79 @@
+//! IO wrappers for Oblivious Linear Function Evaluation (OLE).
+
+#![deny(missing_docs, unreachable_pub, unused_must_use)]
+#![deny(unsafe_code)]
+#![deny(clippy::all)]
+
+use async_trait::async_trait;
+use mpz_common::Context;
+use mpz_fields::Field;
+use mpz_ole_core::OLEError as OLECoreError;
+use mpz_ot::OTError;
+use msg::OLEMessageError;
+use std::{error::Error, fmt::Debug};
+
+pub mod ideal;
+pub mod msg;
+pub mod rot;
+
+/// Batch OLE Sender.
+///
+/// The sender inputs field elements `a_k` and gets outputs `x_k`, such that
+/// `y_k = a_k * b_k + x_k` holds, where `b_k` and `y_k` are the [`OLEReceiver`]'s inputs and outputs
+/// respectively.
+#[async_trait]
+pub trait OLESender<C: Context, F: Field> {
```

Comment:
Let's be consistent with abbreviation across crates

```suggestion
pub trait OLESender<Ctx: Context, F: Field> {
```